### PR TITLE
Renaming NativeReboot to Native , Adding ATS settings and updating the ReadMe

### PR DESCRIPTION
**Pull request by UtpalKumarJha**

### CODE REVIEW

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+@property (nonatomic) BOOL nativeStrandsLoaded,nativeStrandsRendered,nativeStrandsInserted;
+@property (nonatomic,strong) UITableView *tableView;
+@property (nonatomic,strong) NSMutableArray *tableData;
+@end
+@implementation IMNativeViewController
+- (void)viewDidLoad {
+    self.tableData = [[NSMutableArray alloc] init];
+    [self loadInitialData];
+    [self loadInitialData];
+    if(self.isStoryBoard){
```
Comment (prabodhprakash) [inline InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m]: can be written as 
`long long placementId = self.isStoryBoard ? INMOBI_NATIVE_STORYBOARD : INMOBI_NATIVE_MOVIEBOARD;
 self.native = [[IMNative alloc] initWithPlacementId:placementId];`

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+        return cell;
+    } else {
+        IMSimpleTableCell *cell = (IMSimpleTableCell *)[tableView dequeueReusableCellWithIdentifier:@"SimpleTableCell"];
+        cell = [tableView dequeueReusableCellWithIdentifier:@"SimpleTableCell"];
+        if (cell == nil) {
+            NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"SimpleTableCell" owner:self options:nil];
+            cell = [nib objectAtIndex:0];
+            id slide = [self.tableData objectAtIndex:indexPath.row];
+            SampleData *current_slide = slide;
+            cell.Title.text = current_slide.title;
```
Comment (prabodhprakash) [inline InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m]: Please follow camelCasing with all the properties of SampleData

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.h [hunk 3]
```diff
@@ -0,0 +1,15 @@
+//
+//  IMPreRollViewController.h
+//  InMobiAdsDemo
+//
+//  Created by Utpal Kumar Jha on 03/10/17.
```
Comment (prabodhprakash) [inline InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.h]: please remove this.

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.h [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+//  Copyright © 2017 InMobi. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <InMobiSDK/InMobiSDK.h>
+
+@interface IMPreRollViewController : UIViewController
+@property(nonatomic,strong) IMNative *InMobiNativeAd;
```
Comment (prabodhprakash) [inline InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.h]: please follow camelCase.

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.m [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+#import <AVKit/AVKit.h>
+
+
+@interface IMPreRollViewController () <IMNativeDelegate>
+@property (nonatomic) CGFloat screenWidth;
+@property (nonatomic) CGFloat screenHeight;
+@property (nonatomic) CGFloat temp_screen;
+
+@property (weak, nonatomic) IBOutlet UIButton *show_button;
+@property (nonatomic,strong) UIView *PrerollAdView;
```
Comment (prabodhprakash) [inline InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.m]: please use camelCase

File: InMobiAdsDemo/InMobiAdsDemo/CustomCells/IMSimpleTableCell.m [hunk 6]
```diff
@@ -0,0 +1,42 @@
+//
+//  IMSimpleTableCell.m
+//  Ads Demo
+//
+//  Copyright © 2016 Inmobi. All rights reserved.
+//
+
+#import "IMSimpleTableCell.h"
+
+@implementation SampleData
```
Comment (prabodhprakash) [inline InMobiAdsDemo/InMobiAdsDemo/CustomCells/IMSimpleTableCell.m]: not needed

File: InMobiAdsDemo/InMobiAdsDemo/CustomCells/IMSimpleTableCell.m [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+@end
+@implementation IMSimpleTableCell
+@synthesize Title = _Title;
```
Comment (prabodhprakash) [inline InMobiAdsDemo/InMobiAdsDemo/CustomCells/IMSimpleTableCell.m]: synthesize  not needed.

File: InMobiAdsDemo/InMobiAdsDemoTests/InMobiAdsDemoTests.m [hunk 8]
```diff
@@ -0,0 +1,39 @@
+//
```
Comment (prabodhprakash) [inline InMobiAdsDemo/InMobiAdsDemoTests/InMobiAdsDemoTests.m]: no test cases have been mentioned. Please add test cases or remove all test cases related classes, if not needed.

File: InMobiAdsDemo/Podfile [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '9.0'
```
Comment (prabodhprakash) [inline InMobiAdsDemo/Podfile]: is the platform going to be 9.0? we start supporting from 8.0

**Comment by prabodhprakash** [comment]
Please also fix these: 
1. When I show banner ad  (320*50) - there is no way to to back. I had to restart the app.
2. Banner 320*5 with IB - when I click on `expand` in `ad`. I see only a small screen change. Please check the behaviour.
3. Interstitial Full Screen Web View - changes the orientation of the app, i.e. if I load the app in portrait mode and then load above said app - it changes the orientation of the app to landscape.
4. Did not see any ad for "Native Story Boards" and "Native Movie Boards" - please re-verify

File: InMobiAdsDemo/InMobiAdsDemo/AppDelegate.m [hunk 10]
```diff
@@ -0,0 +1,58 @@
+//
+//  AppDelegate.m
+//  InMobiAdsDemo
+//
+//  Created by Tabrez Miyan on 29/09/17.
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/AppDelegate.m]: Done

File: InMobiAdsDemo/InMobiAdsDemo/AppDelegate.m [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+//  Copyright © 2017 InMobi. All rights reserved.
+//
+
+#import "AppDelegate.h"
+#import <InMobiSDK/InMobiSDK.h>
+
+@interface AppDelegate ()
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/AppDelegate.m]: Removed

File: InMobiAdsDemo/InMobiAdsDemo/AppDelegate.m [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    // Override point for customization after application launch.
+    
+    [IMSdk initWithAccountID:@"123"];
+    [IMSdk setLogLevel:kIMSDKLogLevelDebug];
+    [IMSdk getVersion];
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/AppDelegate.m]: Not needed removed

File: InMobiAdsDemo/InMobiAdsDemo/AppDelegate.m [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/AppDelegate.m]: removed

File: InMobiAdsDemo/InMobiAdsDemo/Constants/AppConstants.h [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+//  Copyright (c) 2015 Inmobi. All rights reserved.
+//
+
+#ifndef Ads_Demo_AppConstants_h
+#define Ads_Demo_AppConstants_h
+
+#import "IMAppIds.h"
+
+
+//#define INMOBI_NEWSFEED 1446094659438
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Constants/AppConstants.h]: Done

File: InMobiAdsDemo/InMobiAdsDemo/Constants/AppConstants.h [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+//#define INMOBI_COVERFLOW 1444544814882
+//#define INMOBI_BOARDVIEW 1446178457345
+//#define INMOBI_FEEDS 1444208519758
+//#define INMOBI_BANNER 1447912324502
+//#define INMOBI_INTERSTITIAL 1446377525790
+
+
+#define RGBToUIColor(r, g, b)  [UIColor colorWithRed:((r) / 255.0) green:((g) / 255.0) blue:((b) / 255.0) alpha:1.0]
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Constants/AppConstants.h]: Done

File: InMobiAdsDemo/InMobiAdsDemo/Constants/AppConstants.h [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+
+#define kCellPadding 10.0
+#define kGridAspectRatio (215.0/150.0)
+
+#define kGridItemWidth ((kScreenWidth - (3 * kCellPadding)) / 2)
+#define kGridItemHeight kGridItemWidth * kGridAspectRatio
+
+#define kGridCellSize CGSizeMake (kGridItemWidth, kGridItemHeight)
+
+#define NEWS_URL @"https://api.rss2json.com/v1/api.json?rss_url=http://rss.nytimes.com/services/xml/rss/nyt/World.xml"
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Constants/AppConstants.h]: Done

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMClassicAdViewController.m [hunk 17]
```diff
@@ -0,0 +1,238 @@
...
+            [_adView load];
+
+            [_statusActivityIndicator startAnimating];
+            _statusLabel.hidden = NO;
+            
+            _adViewHeightConstraint.constant = adSize.height;
+            [self.view layoutIfNeeded];
+            
+        } else {
+            if([_adItem objectForKey:@"Type"] && [[_adItem objectForKey:@"Type"] isEqualToString:@"NativeVideo"]){
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMClassicAdViewController.m]: Done

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMClassicAdViewController.m [hunk 18]
```diff
@@ -0,0 +1,238 @@
...
+                _adInterstitial = [[IMInterstitial alloc] initWithPlacementId:INMOBI_INTERSTITIAL_NATIVE_VIDEO delegate:self];
+            }
+            if([_adItem objectForKey:@"Type"] && [[_adItem objectForKey:@"Type"] isEqualToString:@"WebVideo"]){
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMClassicAdViewController.m]: yes Done

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMClassicAdViewController.m [hunk 19]
```diff
@@ -0,0 +1,238 @@
...
+                _adInterstitial = [[IMInterstitial alloc] initWithPlacementId:INMOBI_INTERSTITIAL_RPOSCARD delegate:self];
+            }
+            if([_adItem objectForKey:@"Type"] && [[_adItem objectForKey:@"Type"] isEqualToString:@"NativeDisplay"]){
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMClassicAdViewController.m]: yes Done

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMClassicAdViewController.m [hunk 20]
```diff
@@ -0,0 +1,238 @@
...
+}
+/**
+ * The user will leave application context.
+ */
+-(void)userWillLeaveApplicationFromInterstitial:(IMInterstitial*)interstitial {
+    NSLog(@"%s", __PRETTY_FUNCTION__);
+}
+
+/*
+#pragma mark - Navigation
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMClassicAdViewController.m]: Removed

File: InMobiAdsDemo/InMobiAdsDemo/AppDelegate.h [hunk 21]
```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  InMobiAdsDemo
+//
+//  Created by Tabrez Miyan on 29/09/17.
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/AppDelegate.h]: Removed

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMMainTableViewController.m [hunk 22]
```diff
@@ -0,0 +1,156 @@
...
+                self.preRollVC.placementID = PREROLL_NATIVE;
+                [self.navigationController pushViewController:self.preRollVC animated:true];
+            }
+            break;
+        case 1:
+            if (indexPath.row == 1) {//Expandable Banner
+                [self performSegueWithIdentifier:@"classicAdSegueIB" sender:[types objectAtIndex:indexPath.row]];
+            }
+            else if(indexPath.row == 2){//Native Video
+                [self performSegueWithIdentifier:@"classicAdSegue" sender:[types objectAtIndex:indexPath.row]];
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMMainTableViewController.m]: Yes clubbed into one

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMMainTableViewController.m [hunk 23]
```diff
@@ -0,0 +1,156 @@
...
+
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+    
+    if ([segue.identifier isEqualToString:@"classicAdSegue"]) {
+        IMClassicAdViewController *adVC = segue.destinationViewController;
```
[the remark this answers is not in the document]
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMMainTableViewController.m]: All three are different

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m (hunk 1 above)
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m]: done

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m (hunk 2 above)
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m]: done

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.h (hunk 3 above)
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.h]: done

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.h (hunk 4 above)
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.h]: done

File: InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.m (hunk 5 above)
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/Controllers/IMPreRollViewController.m]: done

File: InMobiAdsDemo/InMobiAdsDemo/CustomCells/IMSimpleTableCell.m (hunk 6 above)
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/CustomCells/IMSimpleTableCell.m]: removed synthesised keywords

File: InMobiAdsDemo/InMobiAdsDemo/CustomCells/IMSimpleTableCell.m (hunk 7 above)
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemo/CustomCells/IMSimpleTableCell.m]: removed

File: InMobiAdsDemo/InMobiAdsDemoTests/InMobiAdsDemoTests.m (hunk 8 above)
Comment (UtpalKumarJha) [reply InMobiAdsDemo/InMobiAdsDemoTests/InMobiAdsDemoTests.m]: Removed all of them as they previously also didn't existed .

**Pull request merged**